Source/WebCore/platform/graphics/ImageUtilities.h

```diff
@@ -44,5 +44,10 @@ WEBCORE_EXPORT Vector<String> findImagesForTranscoding(const Vector<String>& pat
 // happens while transcoding, a null string will be added to the returned list.
 WEBCORE_EXPORT Vector<String> transcodeImages(const Vector<String>& paths, const String& destinationUTI, const String& destinationExtension);
 
+#if PLATFORM(COCOA)
+WEBCORE_EXPORT bool canDataBeDecodedAsSVG(std::span<const uint8_t>);
+WEBCORE_EXPORT std::optional<Vector<FloatSize>> availableSizesFromData(std::span<const uint8_t>, bool isSVGData);
```

Comment:
Given this just returns an empty vector when the boolean `isSVGData` is `true`, why do we bother doing it like this? Doesn’t seem all that helpful to have a function to call just to return an empty vector. How about having the caller only call `availableSizesFromData` when `canDataBeDecodedAsSVG` is `false`?

Reply:
Updated patch to drop `isSVGData` parameter.